docs/clients/promtail/pipelines.md

```diff
@@ -174,7 +207,7 @@ Would create the following `extracted` map:
 
 These map entries can then be used by other pipeline stages such as [timestamp](#timestamp) and/or [output](#output)
 
-[Example in unit test](../../pkg/logentry/stages/regex_test.go)
+[An example can be found in the unit test.](../../pkg/logentry/stages/regex_test.go)
```

Comment:
I'm a bit dubious about referencing source code tests as a good practice of giving examples. What if we do the extra effort to improve the examples in this documentation, without referencing any source code?

Reply:
I agree with this. I'm taking care of the low hanging fruit at the moment but I'll address your feedback here soon.